sets/seaside.py

```diff
@@ -84,6 +85,72 @@ def duration(self):
 		self.game.announce('-- gaining +$1 and +1 action')
 
 
+class Lookout(crd.Card):
+	def __init__(self, game, played_by):
+		crd.Card.__init__(self, game, played_by)
+		self.title = "Lookout"
+		self.price = 3
+		self.description = "{} Look at the top 3 cards of your deck. Trash one of them. " \
+		                   "Discard one of them. Put the other one on the top of your deck.".format(crd.format_actions(1))
+		self.type = "Action"
+
+	@gen.coroutine
+	def play(self, skip=False):
+		crd.Card.play(self, skip)
+		self.played_by.actions += 1
+		self.game.announce('-- gaining 1 action')
+
+		topdeck_list = []
+		for i in range(3):
+			topdeck = self.played_by.topdeck()
+			if topdeck is not None:
+				topdeck_list.append(topdeck)
+
+		empty = False
+		if len(topdeck_list) > 0:
+			to_trash = yield self.played_by.select(1, 1, crd.card_list_to_titles(topdeck_list), 'Choose one to trash')
+			to_trash = self.remove_selection(topdeck_list, to_trash)
+
+			self.played_by.hand.add(to_trash)
+			yield self.played_by.discard([to_trash.title], self.game.trash_pile)
+			self.game.announce('-- trashing {}'.format(to_trash.log_string()))
```

Comment:
don't add the to_trash card to hand unnecessarily you can do `yield self.played_by.discard_floating(...` to discard to the trash pile from outside of hand